src/TSHelper.ts

```diff
@@ -208,10 +208,18 @@ export class TSHelper {
     public static isInTupleReturnFunction(node: ts.Node, checker: ts.TypeChecker): boolean {
         const declaration = TSHelper.findFirstNodeAbove(node, ts.isFunctionLike);
         if (declaration) {
-            let functionType: ts.Type;
+            let functionType: ts.Type | undefined;
             if (ts.isFunctionExpression(declaration) || ts.isArrowFunction(declaration)) {
                 functionType = TSHelper.inferAssignedType(declaration, checker);
-            } else {
+            } else if (ts.isMethodDeclaration(declaration) && ts.isObjectLiteralExpression(declaration.parent)) {
+                // Manually lookup type for object literal properties declared with method syntax
+                const interfaceType = TSHelper.inferAssignedType(declaration.parent, checker);
+                const propertySymbol = interfaceType.getProperty(declaration.name.getText());
```

Comment:
Are these steps required? Can this not be handled the same as functions or arrow functions?

Reply:
`getContextualType` (which is what `inferAssignedType uses`) won't take a method declaration because it's not considered an expression.